src/main/java/uk/gov/hmcts/darts/cases/helper/AdvancedSearchRequestHelper.java

```diff
@@ -134,6 +143,19 @@ private List<Predicate> addCourthouseCriteria(GetCasesSearchRequest request, Cri
         return predicateList;
     }
 
+    private List<Predicate> addPermissionsCriteria(GetCasesSearchRequest request, CriteriaBuilder criteriaBuilder, Root<CourtCaseEntity> caseRoot) {
+        String userAccount = userIdentity.getEmailAddress();
```

Comment:
Do we need this variable - strange name. Probably better to put the RHS down into line 153, since that's the only place that uses this emailAddress.